sonarts-sq-plugin/its/plugin/src/test/java/org/sonar/typescript/its/MissingTSConfigTest.java

```diff
@@ -58,7 +58,8 @@ public void should_work_when_some_files_dont_have_tsconfig() {
     SearchRequest request = new SearchRequest();
     request.setComponentKeys(Collections.singletonList(PROJECT_KEY_MIXED_TSCONFIG)).setRules(ImmutableList.of("typescript:S1764"));
     List<Issue> issuesList = newWsClient().issues().search(request).getIssuesList();
-    assertThat(issuesList).hasSize(2);
+    // TODO should analyze app.ts without tsconfig
```

Comment:
how about adding actual number? `TODO should be 2 (when app.ts is analyzed without tsconfig)`